fr.openfisca.org/legislation-explorer/deploy.sh

```diff
@@ -0,0 +1,7 @@
+git pull
+npm install
+npm install swagger-ui@3.17.3
```

Comment:
Setting this dependency in the legislation explorer would be much cleaner

Reply:
I agree. My thought process was : As long as it is not fixed, we should document the need for the hack. 
I would be interested on your thoughts on this.